imagersite/photo/forms.py

```diff
@@ -0,0 +1,50 @@
+from django import forms
+from photo.models import Photos, Album
+from imagerprofile.models import ImagerProfile
+
+
+class ImagerForm(forms.ModelForm):
+    def __init__(self, user, *args, **kwargs):
+        self.photo_owner = user
+        super(forms.ModelForm, self).__init__(*args, **kwargs)
```

Comment:
Changing the `__init__` semantics of a well-understood Class like the ModelForm is really not a good idea.  You'll end up confusing any other users of your app, who will not expect that the required first argument to a form initializer is not the data for the form to be bound to.